astra-sim/system/Sys.cc

```diff
@@ -1386,6 +1386,10 @@ void Sys::proceed_to_next_vnet_baseline(StreamBaseline* stream) {
     return;
   }
   stream->steps_finished++;
+  // This is hot fix for random failures of simulation.
+  // TODO: Need to find a better way for negative queue id occurrence.
+  if (stream->phases_to_go.front().queue_id < 0)
```

Comment:
Why does it happen? Can we track down the reason?

Reply:
For now, it seems initialization issue. Let me keep track that. 
But for now, it can effectively remove the error.